packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -130,6 +130,8 @@ export const addItemToCart = (payload = {}) => {
             dispatch(toggleDrawer('cart')),
             dispatch(getCartDetails({ forceRefresh: true }))
         ]);
+
+        dispatch(actions.addItem.complete());
```

Comment:
I moved the lines into the try, after the `request` and `receive` so that the cart does not open or update if there was an error. I know you left this comment almost a month ago, but why would the `receive` action go _after_ the `Promise.all`?